docs/RuntimeAttributes.md

```diff
@@ -445,12 +444,6 @@ The following CPU platforms are currently supported by the Google Cloud backend:
 - `Intel Sandy Bridge`
 - `AMD Rome`
 
-### 'useDockerImageCache'
```

Comment:
Do we know for sure this isn't supported in Batch? The "only PAPI v2 beta" here means that previous google products don't support it, the author of this comment didn't know about Batch.

Reply:
When I was looking through the tests I saw this:
> backends: [Papiv2-Docker-Image-Cache, GCPBATCH_DOCKER_IMAGE_CACHE_NOT_IMPLEMENTED]
and no references to this runtime attribute other than in those specific tests